core/arch/arm/kernel/secure_partition.c

```diff
@@ -369,19 +373,20 @@ static TEE_Result stmm_enter_invoke_cmd(struct tee_ta_session *s,
 
 	res = sec_part_enter_user_mode(spc);
 	if (res)
-		goto out;
+		goto err_out;
 	/*
-	 * Copy the SPM response from secure partition back to the non secure
-	 * the client that called us
+	 * Copy the SPM response from secure partition back to the non-secure
+	 * the client that called us.
 	 */
 	param->u[1].val.a = spc->regs.x[4];
 
 	memcpy(va, (void *)spc->ns_comm_buf_addr, ns_buf_size);
 
-out:
+err_out:
```

Comment:
This is not a specific error path.
Prefer rename `err_out` to `out_session` and `err_va` to `out_va`.